### ISSUE TITLE
Summarize dataset:   0%|          | 0/19 [00:00<?, ?it/s]

### ISSUE THREAD

**Issue by PranavMare** [pandas-profiling/pandas-profiling#806]
I was just getting started with Pandas Profiling however, I encounter a problem. The script was running but the profile report is not being generated for some reason. Can anyone let me know if it's a bug or I am doing something wrong? 

![image](https://user-images.githubusercontent.com/56601384/125128139-96ec8d00-e0cb-11eb-8f51-0af30ae47cde.png)

**Comment by davidcereal** [comment]
Same issue.

**Comment by dishashetty** [comment]
@PranavMare  @davidcereal were you able to resolve this? I am facing the same issue.

**Comment by PranavMare** [comment]
@dishashetty Yes I was able to resolve it back then. I don't remember exactly what I did. But I think you need to restart the kernel every time your run the code block since pandas profiling relies on other dependencies which makes it necessary to restart the kernel.
